recipes/featuretools_sql/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "featuretools_sql" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/featuretools_sql-{{ version }}.tar.gz
+  sha256: 46d090103f7721958a940a10b1a7a40a385278b802134f9ca04ed90d46a45291
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8.*
+    - pip
+  run:
+    - python >=3.6,<4.0
+    - numpy >=1.17.5
+    - pandas >=1.3.0
+    - featuretools >=1.5.0
+    - psycopg2 >=2.9.3
+    - connectorx >=0.3.0
```

Comment:
remove or change?

Reply:
Yep, updating now. `connectorX` will be removed, replaced by `sqlalchemy` and `pymysql`